docs/the_black_code_style/current_style.md

```diff
@@ -233,8 +233,10 @@ _Black_ prefers double quotes (`"` and `"""`) over single quotes (`'` and `'''`)
 will replace the latter with the former as long as it does not result in more backslash
 escapes than before.
 
-_Black_ also standardizes string prefixes, making them always lowercase. On top of that,
-if your code is already Python 3.6+ only or it's using the `unicode_literals` future
+_Black_ also standardizes string prefixes. Prefix characters are made lowercase with the
+exception of [capital "R" prefixes](#rstrings-and-rstrings), and in the case of multiple
+characters "r" is put first as in spoken language: "raw f-string". On top of that, if
+your code is already Python 3.6+ only or it's using the `unicode_literals` future
```

Comment:
This should be changed (somewhat unrelated to this PR): we only support formatting 3.6+, so we currently always remove the `u`.

Reply:
Good catch 👍 We still support 3.3-3.5 right? Not that it changes the message but anyways.

Reply:
Oh right, we do. But I'm not sure what the "3.6+" is about in the first place in that case, since u prefixes are redundant in all of Python 3. (And in Python 3.0-3.2 they did nothing.)